scripts/.storybook/osano.css

```diff
@@ -0,0 +1,220 @@
+@import '~@contentful/forma-36-tokens/dist/css/index.css';
```

Comment:
can't we use `@import '~@contentful/experience-consent-manager/dist/style.css'`?

Reply:
@suevalov `@contentful/experience-consent-manager` is is like a separate package? I can not find it 😬

Reply:
yep, https://github.com/contentful/creator-app-mvp/blob/master/package.json#L54

Reply:
@suevalov it worked, just needed to overwrite some stuff that was overwritten in the  `experience-consent-manager` 🤪